influxdb2_client/src/models/bucket.rs

```diff
@@ -0,0 +1,114 @@
+//! Bucket
+
+use serde::{Deserialize, Serialize};
+
+/// Bucket Schema
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct Bucket {
+    /// BucketLinks
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub links: Option<crate::models::BucketLinks>,
+    /// Bucket ID
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub id: Option<String>,
+    /// Bucket Type
+    #[serde(rename = "type", skip_serializing_if = "Option::is_none")]
+    pub r#type: Option<Type>,
+    /// Bucket name
+    pub name: String,
+    /// Bucket description
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub description: Option<String>,
+    /// Organization ID of bucket
+    #[serde(rename = "orgID", skip_serializing_if = "Option::is_none")]
+    pub org_id: Option<String>,
+    /// RP
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub rp: Option<String>,
+    /// Created At
+    #[serde(rename = "createdAt", skip_serializing_if = "Option::is_none")]
+    pub created_at: Option<String>,
+    /// Updated At
+    #[serde(rename = "updatedAt", skip_serializing_if = "Option::is_none")]
+    pub updated_at: Option<String>,
+    /// Rules to expire or retain data. No rules means data never expires.
+    #[serde(rename = "retentionRules")]
+    pub retention_rules: Vec<crate::models::RetentionRule>,
+    /// Bucket labels
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub labels: Option<Vec<crate::models::Label>>,
```

Comment:
So `Option<Vec<T>>` is kind of a rarely used type in Rust because you can just have an empty `Vec`, as long as an empty `Vec` doesn't have a different meaning from `None`. Looking at the swagger definition of `Bucket` and some of the API calls that use `Bucket`, I don't think there is a difference? Do you know of any reason this and other places with `Option<Vec<T>>` couldn't be:

```suggestion
    #[serde(default, skip_serializing_if = "Vec::is_empty")]
    pub labels: Vec<crate::models::Label>,
```

Reply:
I believe you are right, This looks to me like a better solution

Reply:
I have updated all instances of Option<Vec<T>>